src/MarkdownTextInput.web.tsx

```diff
@@ -645,6 +666,8 @@ const styles = StyleSheet.create({
     overflowY: 'auto',
     overflowX: 'auto',
     overflowWrap: 'break-word',
+    position: 'relative',
+    clipPath: 'polygon(0% 0%, 100% 0%, 100% 100%, 0% 100%)',
```

Comment:
Because we are positioning the `pre` background absolutely it can actually overflow the contenteditable div. Due to the overflow being set to 'auto' it can be displayed outside of the div. 

Since we can't change the overflow type the ideal solution is to add a clipPath to the input so that it gets cut off

Reply:
Nice, thanks for explanation :D